<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatPage</class>
 <widget class="QWidget" name="ChatPage">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>iutgram</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#e0e0e0;
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>91</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{
border-top-right-radius:10px;
border-bottom-right-radius:10px;
background-color:#324D6A;
padding: 20px;
}
</string>
   </property>
   <widget class="QPushButton" name="btn_join_channel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>join channel</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border : none;
outline: none;
padding:10px;
color : white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="source.qrc">
      <normaloff>:/src/img/chanel+.png</normaloff>:/src/img/chanel+.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_new_chat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>207</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>new chat</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border : none;
outline: none;
padding:10px;
color : white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="source.qrc">
      <normaloff>:/src/img/chat.png</normaloff>:/src/img/chat.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_logout">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>500</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>logout</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border : none;
outline: none;
padding:10px;
color : white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="source.qrc">
      <normaloff>:/src/img/out.png</normaloff>:/src/img/out.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_new_group">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>create group</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border : none;
outline: none;
padding:10px;
color : white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="source.qrc">
      <normaloff>:/src/img/grup.png</normaloff>:/src/img/grup.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_new_channel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>394</y>
      <width>48</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>create channel</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border : none;
outline: none;
padding:10px;
color : white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="source.qrc">
      <normaloff>:/src/img/chanel.png</normaloff>:/src/img/chanel.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_join_group">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>122</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>join group</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border : none;
outline: none;
padding:10px;
color : white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="source.qrc">
      <normaloff>:/src/img/grup+.png</normaloff>:/src/img/grup+.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="chatsList">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>541</width>
     <height>461</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScrollMargin">
    <number>16</number>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>541</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#324D6A;
border-top-right-radius:20px;
border-top-left-radius:20px;</string>
   </property>
   <widget class="QLabel" name="chat_title">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:18px;
font-weight:bold;
color:white;</string>
    </property>
    <property name="text">
     <string>blank</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border : none;
outline: none;
padding:10px;
color : white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="source.qrc">
      <normaloff>:/src/img/person.png</normaloff>:/src/img/person.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>520</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#324D6A;
border-bottom-right-radius:20px;
border-bottom-left-radius:20px;</string>
   </property>
   <widget class="QLineEdit" name="input_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
color:black;
font-size:16px;
border-radius:5px;
padding:10px;</string>
    </property>
    <property name="placeholderText">
     <string>Type your message . . .</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sendMessage">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="source.qrc">
      <normaloff>:/src/img/send_icon.png</normaloff>:/src/img/send_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>291</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
outline:none;
background-color:white;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_chats">
    <attribute name="title">
     <string>personal</string>
    </attribute>
    <widget class="QListWidget" name="messagesList_chat">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_channels">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>channels</string>
    </attribute>
    <widget class="QListWidget" name="messagesList_channel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_group">
    <attribute name="title">
     <string>groups</string>
    </attribute>
    <widget class="QListWidget" name="messagesList_group">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_scrollBottom">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>470</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
border : none;
outline: none;
padding:10px;
color : white;
opacity:0.5;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/src/img/44567.png</normaloff>:/src/img/44567.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_con_status">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255, 74, 74,0.6);
color:white;
font-weight:bold;
padding:5px;</string>
   </property>
   <property name="text">
    <string>Your Connection Status : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
